.github/workflows/elixir_test.yml

```diff
@@ -0,0 +1,55 @@
+name: Elixir CI
+
+on:
+  workflow_call:
+
+env:
+  CACHE_NAME_DEPS: cache-elixir-deps
+  CACHE_NAME_COMPILED_TEST: cache-compiled-test-build
+  ELIXIR_ASSERT_TIMEOUT: 1000
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-20.04
```

Comment:
Bumping to 24.04? Or does it cause problems?

Reply:
I do want to keep changes as small as possible, therefore will go with ubuntu-20.04 in the first step